Okta.Core.Tests/Clients/Helpers.cs

```diff
@@ -155,5 +156,31 @@ internal static Models.Group CreateGroup(OktaClient oktaClient, TestGroup dbGrou
             return oktaGroup;
         }
 
+        internal static string GetRandomString()
+        {
+            const string AllowedChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz#@$^*()";
+            Random rng = new Random();
+
+            return RandomString(AllowedChars, 8, 16, rng);
+        }
+
+        private static string RandomString(
+            string allowedChars,
+            int minLength,

```

Comment:
Fair point. Probably an overkill but I was looking for a function that would generate a random string quick and dirty (since it's only used in the context of password change unit test and this code popped up. Fine with simplifying/refactoring this at a later point though.